README.md

```diff
@@ -246,9 +246,10 @@ conn.on(:stream) do |stream|
     }
 
     # initiate server push stream
+    push_stream = nil
     stream.promise(head) do |push|
-      push.headers({ ... })
-      push.data(...)
```

Comment:
Sorry about the confusion, I meant that we should keep .headers({...}) in place to demonstrate how to send additional headers. the .data() calls can happen in the callback or later -- this is up to the application to decide based on what it's pushing, etc.